src/MultiReport/Mouse.cpp

```diff
@@ -26,74 +26,45 @@ THE SOFTWARE.
-  D_REPORT_SIZE, 0x08,                            //     REPORT_SIZE (8)
-  D_REPORT_COUNT, 0x02,                           //     REPORT_COUNT (3)
-  D_INPUT, (D_DATA|D_VARIABLE|D_RELATIVE),        //     INPUT (Data,Var,Rel)
-
-  /* Wheels */
-
-  /* Vertical wheel */
-  D_COLLECTION, D_LOGICAL,                        //     COLLECTION (Logical)
-  D_USAGE, 0x48,                                  //      USAGE (Resolution Multiplier)
-  D_LOGICAL_MINIMUM, 0x00,                        //       LOGICAL_MINIMUM (0)
-  D_LOGICAL_MAXIMUM, 0x01,                        //       LOGICAL_MAXIMUM (1)
-  D_PHYSICAL_MINIMUM, 0x01,                       //       PHYSICAL_MINIMUM (1)
-  D_PHYSICAL_MAXIMUM, 0x04,                       //       PHYSICAL_MAXIMUM (4)
-  D_REPORT_SIZE, 0x02,                            //       REPORT_SIZE (2)
-  D_REPORT_COUNT, 0x01,                           //       REPORT_COUNT (1)
-  D_PUSH,                                         //       PUSH
-  D_FEATURE, (D_DATA|D_VARIABLE|D_ABSOLUTE),      //       FEATURE (Data,Var,Abs)
-  D_USAGE, 0x38,                                  //      USAGE (Wheel)
-  D_LOGICAL_MINIMUM, 0x81,                        //       LOGICAL_MINIMUM (-127)
-  D_LOGICAL_MAXIMUM, 0x7f,                        //       LOGICAL_MAXIMUM (127)
-  D_PHYSICAL_MINIMUM, 0x00,                       //       PHYSICAL_MINIMUM (0)
-  D_PHYSICAL_MAXIMUM, 0x00,                       //       PHYSICAL_MAXIMUM (0)
-  D_REPORT_SIZE, 0x08,                            //       REPORT_SIZE (8)
-  D_INPUT, (D_DATA|D_VARIABLE|D_RELATIVE),        //       INPUT (Data,Var,Rel)
-  D_END_COLLECTION,                               //     END_COLLECTION
+  D_USAGE_PAGE, D_PAGE_BUTTON,                      /*     USAGE_PAGE (Button) */
+  D_USAGE_MINIMUM, 0x01,                      /*     USAGE_MINIMUM (Button 1) */
+  D_USAGE_MAXIMUM, 0x08,                      /*     USAGE_MAXIMUM (Button 8) */
+  D_LOGICAL_MINIMUM, 0x00,                      /*     LOGICAL_MINIMUM (0) */
+  D_LOGICAL_MAXIMUM, 0x01,                      /*     LOGICAL_MAXIMUM (1) */
+  D_REPORT_COUNT, 0x08,                      /*     REPORT_COUNT (8) */
+  D_REPORT_SIZE, 0x01,                      /*     REPORT_SIZE (1) */
+  D_INPUT, (D_DATA|D_VARIABLE|D_ABSOLUTE),                      /*     INPUT (Data,Var,Abs) */
+
+  /* X, Y, Wheel */
+  D_USAGE_PAGE, D_PAGE_GENERIC_DESKTOP,                      /*     USAGE_PAGE (Generic Desktop) */
+  D_USAGE, 0x30,                      /*     USAGE (X) */
+  D_USAGE, 0x31,                      /*     USAGE (Y) */
+  D_USAGE, 0x38,                      /*     USAGE (Wheel) */
+  D_LOGICAL_MINIMUM, 0x81,                      /*     LOGICAL_MINIMUM (-127) */
+  D_LOGICAL_MAXIMUM, 0x7f,                      /*     LOGICAL_MAXIMUM (127) */
+  D_REPORT_SIZE, 0x08,                      /*     REPORT_SIZE (8) */
+  D_REPORT_COUNT, 0x03,                      /*     REPORT_COUNT (3) */
```

Comment:
This line looks wrong. or rather, the code changed from master, but not the comment. one of them _is_ wrong

Reply:
0x03 is the good count *now*, 0x02 was good in the *previous* setup.

Originally, we had `X`, `Y`, and `VWheel` described together, thus `REPORT_COUNT(3)`. The previous descriptor change (#19) split the `VWheel` out of these, and `REPORT_COUNT(2)` was the correct size there (but the comment remained 3, and was therefore incorrect). With the revert, `VWheel` is described along the axes again, and `REPORT_COUNT(3)` is the good size.

So the new code is correct, the comment was previously wrong, but is correct now.